assets/src/edit-story/components/keyboardShortcutsMenu/index.js

```diff
@@ -18,79 +18,92 @@
-import Modal from '../modal';
-import { Keyboard as KeyboardShortcutsButton } from '../button';
+import {
+  Button,
+  BUTTON_SIZES,
+  BUTTON_TYPES,
+  BUTTON_VARIANTS,
+  Icons,
+  useGlobalKeyDownEffect,
+  useFocusOut,
+  Tooltip,
+  TOOLTIP_PLACEMENT,
+} from '../../../design-system';
+import { useConfig } from '../../app';
+import { Popup } from './popup';
 import ShortcutMenu from './shortcutMenu';
 import { TOGGLE_SHORTCUTS_MENU } from './constants';
 
-function KeyboardShortcutsMenu({ onMenuToggled }) {
+const Wrapper = styled.div``;
+
+const withRTLPlugins = [stylisRTLPlugin];
+const withoutRTLPlugins = [];
+
+function KeyboardShortcutsMenu() {
   const anchorRef = useRef();
+  const wrapperRef = useRef();
   const [isOpen, setIsOpen] = useState(false);
+  const { isRTL } = useConfig();
 
-  const toggleMenu = useCallback(
-    (e, showMenu) => {
-      e.preventDefault();
+  const closeMenu = useCallback(() => setIsOpen(false), [setIsOpen]);
 
-      setIsOpen((prevIsOpen) => {
-        const menuOpen = showMenu ?? !prevIsOpen;
+  const toggleMenu = useCallback((e, showMenu) => {
+    e.preventDefault();
+    setIsOpen((prevIsOpen) => {
+      const menuOpen = showMenu ?? !prevIsOpen;
 
-        if (onMenuToggled) {
-          onMenuToggled(menuOpen);
-        }
+      if (isKeyboardUser() && !menuOpen) {
+        // When menu closes, return focus to toggle menu button
+        anchorRef.current.focus?.();
+      }
 
-        if (isKeyboardUser() && !menuOpen) {
-          // When menu closes, return focus to toggle menu button
-          anchorRef.current.focus?.();
-        }
+      return menuOpen;
+    });
+  }, []);
 
-        return menuOpen;
-      });
-    },
-    [onMenuToggled]
-  );
+  const label = __('Open Keyboard Shortcuts', 'web-stories');
 
   useGlobalKeyDownEffect(TOGGLE_SHORTCUTS_MENU, toggleMenu, [toggleMenu]);
+  useFocusOut(wrapperRef, closeMenu, []);
 
   return (
-    <>
-      <WithTooltip
-        title={__('Open Keyboard Shortcuts', 'web-stories')}
-        placement={Placement.TOP}
-      >
-        <KeyboardShortcutsButton
-          ref={anchorRef}
-          width="24"
-          height="24"
-          aria-pressed={isOpen}
-          aria-haspopup={true}
-          aria-expanded={isOpen}
-          aria-label={__('Open Keyboard Shortcuts', 'web-stories')}
-          onClick={toggleMenu}
-        />
-      </WithTooltip>
-      <Modal
-        contentLabel={__('Keyboard Shortcuts Menu', 'web-stories')}
-        open={isOpen}
-        onClose={toggleMenu}
-      >
-        <ShortcutMenu toggleMenu={toggleMenu} />
-      </Modal>
-    </>
+    <StyleSheetManager
+      stylisPlugins={isRTL ? withRTLPlugins : withoutRTLPlugins}
+    >
+      <Wrapper ref={wrapperRef}>
+        <Tooltip
+          title={label}
+          placement={TOOLTIP_PLACEMENT.TOP}
+          shortcut="mod+/"
+          hasTail
+        >
+          <Button
+            ref={anchorRef}
+            variant={BUTTON_VARIANTS.SQUARE}
```

Comment:
The "Circle" part of the layer name confused me a bit too at first. This is actually a square button - the circle button doesn't exist at all in Figma.

Compare the green circles below with the red circle. I think the "Circle" label in the red one just refers to the fact, that the button per default is showing a Circle icon in the design system:

![Screen Shot 2021-02-16 at 10 29 31](https://user-images.githubusercontent.com/637548/108084278-0345f180-7042-11eb-8ef3-8296984b051a.png)

Reply:
I think the circle was added for some edge cases, but not completely sure... Maybe @BrittanyIRL can weigh in?

Reply:
🕸️   Back in November, figma was updated from 'circle' to 'square' variants (https://github.com/google/web-stories-wp/pull/5245/files) but I kept the circular version because I felt both would come in handy. 

(I can't find any slack comms about this update to further explain it)